Makefile

```diff
@@ -0,0 +1,24 @@
+# Copyright SecureKey Technologies Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+GO_CMD ?= go
+GO111MODULE=on
+
+all: checks unit-test
+
+checks: license lint
+
+lint:
+	@scripts/check_lint.sh
+
+license:
```

Comment:
PHONY tag is missing for several targets.

Reply:
Added PHONY tag - https://github.com/hyperledger/aries-framework-go/compare/ffab3ad151e63cb395efa46935e28bfa1849f589..6a349ceee3704e5d4a1d8da0a776e7e4d5b83c5f